docusaurus.config.js

```diff
@@ -144,26 +144,104 @@ module.exports = {
+            },
+            {
+              html: `<a href="https://twitter.com/temporalio" aria-label="Twitter"><svg xmlns="http://www.w3.org/2000/svg" class="footer__svg" width="16" height="16" viewBox="0 0 24 24"><path d="M24 4.557c-.883.392-1.832.656-2.828.775 1.017-.609 1.798-1.574 2.165-2.724-.951.564-2.005.974-3.127 1.195-.897-.957-2.178-1.555-3.594-1.555-3.179 0-5.515 2.966-4.797 6.045-4.091-.205-7.719-2.165-10.148-5.144-1.29 2.213-.669 5.108 1.523 6.574-.806-.026-1.566-.247-2.229-.616-.054 2.281 1.581 4.415 3.949 4.89-.693.188-1.452.232-2.224.084.626 1.956 2.444 3.379 4.6 3.419-2.07 1.623-4.678 2.348-7.29 2.04 2.179 1.397 4.768 2.212 7.548 2.212 9.142 0 14.307-7.721 13.995-14.646.962-.695 1.797-1.562 2.457-2.549z"/></svg> Twitter</a>`,
+            },
+            {
+              html: `<a href="https://www.youtube.com/channel/UCGovZyy8OfFPNlNV0i1fI1g" aria-label="YouTube"><svg xmlns="http://www.w3.org/2000/svg" class="footer__svg" width="16" height="16" viewBox="0 0 24 24"><path d="M19.615 3.184c-3.604-.246-11.631-.245-15.23 0-3.897.266-4.356 2.62-4.385 8.816.029 6.185.484 8.549 4.385 8.816 3.6.245 11.626.246 15.23 0 3.897-.266 4.356-2.62 4.385-8.816-.029-6.185-.484-8.549-4.385-8.816zm-10.615 12.816v-8l8 3.993-8 4.007z"/></svg> YouTube</a>`,
+            },
+          ],
+        },
+        {
+          items: [
+            {
+              label: "Meetups",
+              href: "https://lu.ma/temporal",
+            },
+            {
+              label: "Workshops",
+              href: "https://temporal.io/community#workshops",
+            },
+            {
+              label: "Support Forum",
+              href: "https://community.temporal.io/",
+            },
+          ],
+        },
+        {
+          items: [
+            {
+              label: "Use Cases",
+              href: "https://temporal.io/use-cases",
+            },
+            {
+              label: "Case Studies",
+              href: "http://localhost:3000/blog/tags/case-study",
+            },
+            {
+              label: "Blog",
+              to: "/blog",
+            },
+          ],
+        },
+        {
+          items: [
+            {
+              label: "About the docs",
+              href: "https://github.com/temporalio/documentation/blob/master/README.md",
+            },
+            {
+              label: "Security",
+              to: "/security",
+            },
+            {
+              label: "Privacy Policy",
+              to: "/privacy-policy",
+            },
+            {
+              label: "Terms of Service",
+              href: "https://docs.temporal.io/pdf/temporal-tos-2021-07-24.pdf",
+            },
+          ],
+        },
+        {
+          items: [
+            {
+              label: "Join The Cloud Waitlist",
```

Comment:
```suggestion
              label: "Join the Cloud Waitlist",
```
Although you might want to stet, at least for now, if you want to match the home page.